lib/pluginmanager/templates/filter-plugin/lib/logstash/filters/example.rb.erb

```diff
@@ -0,0 +1,43 @@
+# encoding: utf-8
+require "logstash/filters/base"
+require "logstash/namespace"
+
+# This <%= @plugin_name %> filter will replace the contents of the default 
+# message field with whatever you specify in the configuration.
+#
+# It is only intended to be used as an <%= @plugin_name %>.
+class LogStash::Filters::<%= classify(@plugin_name) %> < LogStash::Filters::Base
```

Comment:
We are using the `classify` a few times in the templates, might be a good idea to provide the templates with a `plugin_class` in the render context?